.htmlvalidate.json

```diff
@@ -8,6 +8,7 @@
   },
 
   "rules": {
+    "attribute-allowed-values": "warn",
```

Comment:
We should avoid downgrading existing errors to warnings, and instead find a solution that does not produce errors.